docs/cvl/builtin.md

```diff
@@ -173,3 +174,32 @@ instruments the contract code at `p` to set `x_p` to `true`.  The Prover then
 tries to prove that `x_p` is false after executing the function.  To find a
 counterexample; the Prover must construct a model that passes through `p`.
 
+(built-in-readonly-reentrancy)=
+Read-only reentrancy detection &mdash; `readOnlyReentrancy`
+-----------------------------------------------------------
+
+The `readOnlyReentrancy`  built-in rule detects 
+[read-only reentrancy vulnerabilities in a contract][readonly-reentrancy-vulnerability].
+
+[readonly-reentrancy-vulnerability]: https://blog.pessimistic.io/read-only-reentrancy-in-depth-6ea7e9d78e85
+
+The `readOnlyReentrancy` rule can be enabled by including
+```cvl
+use builtin rule readOnlyReentrancy;
```

Comment:
I think that it's still viewReentrancy. I can merge  the change if we think it's needed it but I didn't want to do that before the conferences